src/server/scripts/EasternKingdoms/ScarletMonastery/instance_scarlet_monastery.cpp

```diff
@@ -315,52 +356,57 @@ class npc_mograine : public CreatureScript
             switch (uiSteps)
             {
                 case 1:
-                    me->GetMotionMaster()->MovePoint(0, 1152.039795f, 1398.405518f, 32.527878f);
+                    me->SetFacingTo(3.13f);
```

Comment:
this is oddly specific

Reply:
In the new implementation (hmm, in the future I'll try to avoid rebase-editing & force-pushing and use WIP commits that I squash later, as we can't see the change here 😔) I have changed this to be so that Mograine turns to face the chapel doors. What I need him to do is face directly south. I guess I could use some pi constant for this as well? Which do you think is a better approach?

Reply:
Nothing wrong with oddly specific :P Just curious where it comes from.

We have more oddly specific ones, see
https://github.com/azerothcore/azerothcore-wotlk/blob/5367eb4b1dfb98016e561e4c9a3a06d5efd4e4f1/src/server/scripts/EasternKingdoms/Scholomance/boss_kirtonos_the_herald.cpp#L149C22-L149C22

Reply:
> We have more oddly specific ones, see [Kirtonos]

Wonder if that's sniffed.

Reply:
This is supposed to be facing the player, and I have a video of the Ashbringer dialogue triggered from the side, which was recorded before the Chinese server was shut down, and in the video, you can see Mograine kneeling facing the player

Reply:
> Nothing wrong with oddly specific :P Just curious where it comes from.

Fair enough 😅 I think the chapel door was maybe a slight improvement, got rid of a magic number 👀

> We have more oddly specific ones, see [Kirtonos]

Okay, yeah, that's very specific :D

> This is supposed to be player-facing, I have a side-triggered video of Ashbrlinger dialogue recorded before the Chinese server shuts down, in which "Moglean" can be seen kneeling facing the player. (WLK third-party servers use spell SPELL_FORGIVENESS and in the video SPELL_COSMETIC_CHAIN killing Mowglien has no lightning strike effect on the target, while in the video, Mowglion has a lightning effect on his body when he is killed, and spell 28282 is resident as a buff.)
> https://clipchamp.com/watch/xhyWRIV91X8

Excellent catch on Commander Mograine turning to face the player and not south 💎 I was able to verify this from my source videos, linked in the PR description, as well. I have applied this change.

As for Highlord Mograine using the wrong spell - I'll put it on my todo list and fix it at some point hopefully.

Reply:
I pointed out that the curse had a lightning effect to prove that the video was recorded on an official Blizzard server, not an unofficial server